.github/workflows/Tests.yml

```diff
@@ -136,10 +130,10 @@ jobs:
     defaults:
       run:
         shell: bash
-    timeout-minutes: 10
+    timeout-minutes: 8
```

Comment:
@coderabbitai, please check the average duration of all jobs recently and confirm if 8 minutes should still be enough time.